### DISCUSSION TITLE
Select

### DISCUSSION THREAD

**Discussion by andelkocvjetkovic** [adobe/react-spectrum#3276] [Q&A]
Hello, is it possible to render Select component from [the docs ](https://react-spectrum.adobe.com/react-aria/useSelect.html) with empty items, and have a placeholder that says no items to show or something like that.

**Comment by andelkocvjetkovic** [comment]
like this
https://stackblitz.com/edit/vitejs-vite-5twuug?file=src/App.jsx

**Reply by snowystinger** [reply]
Yes and no, we supported Disabled items, they will not be the placeholder if no options are selected and they are the only item. https://react-spectrum.adobe.com/react-spectrum/Picker.html#props (see placeholder)
However, the example you supplied is a bit of a weird case, why should it do that? I think it'd be better to supply a placeholder. If you have any disabled values, those are fine, they aren't selectable, which makes sense. And you can see this if you make a native picker with both disabled and non-disabled options
```
      <select>
        <option value disabled>
          No item to show
        </option>
        <option value>First Item</option>
      </select>
```
You cannot select the disabled one, and the select doesn't render it as the placeholder anymore. It renders First Item instead.

**Reply by andelkocvjetkovic** [reply]
Yea, tnx for answer it helped me. Cheers 🍻
